AppCenterReactNativeShared/Products/local.podspec

```diff
@@ -3,7 +3,7 @@
 
 Pod::Spec.new do |s|
   s.name              = 'AppCenterReactNativeShared'
-  s.version           = '4.4.3'
+  s.version           = '4.4.1'
```

Comment:
Why? I believe it should be 4.4.3.
as everywhere:

https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec#L3
https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-analytics/package.json#L3
https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-crashes/package.json#L3
https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter/package.json#L3

Reply:
@AnatolyPristensky could you please explain?

Reply:
@DmitriyKirakosyan leaving it on 4.4.3 causing the following error on CI: 
```
CocoaPods could not find compatible versions for pod "AppCenterReactNativeShared":
  In snapshot (Podfile.lock):
    AppCenterReactNativeShared (from `../../AppCenterReactNativeShared/Products/local.podspec`)
  In Podfile:
    AppCenterReactNativeShared (from `../../AppCenterReactNativeShared/Products/local.podspec`)
```

I'm not sure why, but looks like that local.podspec should contain version of Apple SDK.

Reply:
@AnatolyPristensky  local.podspec should not be used on CI. It is for local build only. Please check why it gets used on CI.